src/Tizen.Multimedia.MediaPlayer/Player/Player.cs

```diff
@@ -549,15 +551,17 @@ public int GetPlayPosition()
             return playPosition;
         }
 
-        private void SetPlayPosition(int milliseconds, bool accurate,
+        private void SetPlayPosition(long position, bool accurate, bool nanoseconds,
```

Comment:
Don't we need to consider extensibility on new unit?
(Is there any possibility to add new unit in the future? )

Reply:
i think not. but if then, i'll make new enum.